setup.py

```diff
@@ -36,7 +36,10 @@
         'boto3>=1.9.131',
         'clint',
         'couchdb-cluster-admin>=0.5.0',
-        'cryptography>=2.3',  # security update
+	# 2.8 solves issue on MacOS Catalina
+	# https://github.com/pypa/pip/issues/7254
+	# but 2.9 fails to compile
```

Comment:
weird indentation (tabs?!)

Reply:
Ugh, weird. New computer. Must have done this before I'd set up my normal text editor configurations

Reply:
https://github.com/dimagi/commcare-cloud/pull/3758